src/firefly/js/templates/common/ttFeatureWatchers.js

```diff
@@ -71,12 +71,9 @@ function setupObsCorePackaging(tbl_id) {
 }
 
 function updateSearchRequest( tbl_id='', dlParams='', sRequest=null) {
-    const hostname= new URL(sRequest.source)?.hostname;
-
+    const hostname= new URL(sRequest.source ? sRequest.source : sRequest.serviceUrl)?.hostname;
```

Comment:
actually I can just simply this to `sRequest.source ?? sRequest.serviceUrl`. I'll add that in tomorrow, but let me know if it looks good & if testing it works for both CADC / DCE @robyww.